### PR TITLE
2109 localwork

### PR DESCRIPTION
**Pull request by Adityapadmanair**
This is the newly created branch with the Bokeh issue. Please take a look.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,19 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "netcdfjs": "^3.0.0"
+    "@bokeh/bokehjs": "^3.2.2",
+    "@bokeh/numbro": "^1.6.2",
+    "@bokeh/slickgrid": "^2.4.4102",
+    "axios": "^1.5.0",
+    "bokehjs": "^1.4.0",
```
Comment (pastewka) [inline package.json:21]: This dependency should be removed

File: src/main.js [hunk 2]
```diff
@@ -1,8 +1,103 @@
+import {Palettes} from '@bokeh/bokehjs';
+import * as Bokeh from 'bokehjs';
```
Comment (pastewka) [inline src/main.js:2]: You don't need this import. 'bokehjs' is not the Bokeh plotting library.

File: src/main.js [hunk 3]
```diff
@@ -1,8 +1,103 @@
...
+            for (let y = 0; y < ySize; y++) {
+              for (let x = 0; x < xSize; x++) {
+                const value = heights[y * xSize + x];
+                const scaledValue = (value - minValue) / (maxValue - minValue);
+                const colorIndex = Math.floor(scaledValue * (colorPalette.length - 1));
+                const color = colorPalette[colorIndex];
+
+                console.log(`x: ${x}, y: ${y}, value: ${value}, scaledValue: ${scaledValue}, color: ${color}`);
+                
+                ctx.fillStyle = color;
```
Comment (pastewka) [inline src/main.js:86]: `color` is an integer. You need to turn that into a HTML color code: `ctx.fillStyle = '#'+(color.toString(16).substring(0, 6));`

**Review by pastewka — CHANGES_REQUESTED** [review]

**Comment by Adityapadmanair** [comment]
Doing a new pull req with the above mentioned changes

**Pull request closed without merging**